### ISSUE TITLE
Thank you

### ISSUE THREAD

**Issue by strogiyotec** [paoloap/zaread#3]
I spend an hour searching for something like this. As you do I hate this bloated MS Office interface, it's awesome that zathura is used as default pdf reader because all other readers suck. Anyway thank you for this wonderful script

**Comment by PeSader** [comment]
Yep, I wanna second this! ```zaread``` is an amazing one-of-a-kind script!

Thanks a lot :)

**Comment by paoloap** [comment]
Well guys, I'm really embarassed that I didn't check my github notifications for some years and didn't know that so many people adopted this script... I'm really glad that you appreciated it, maybe it's time to put my hands again on it and merge all those pull requests I received

**Comment by fossimprove** [comment]
Thank you, came from reddit to say here!

Also I would like to just make simple suggestion to everyone to try [Sioyek](https://github.com/ahrm/sioyek) which is actually great and could make your productive workflow better!

**Comment by kofm** [comment]
Thank you so much!

**Comment by mrpingo** [comment]
Brilliant!
using it two years now

Thank you!
